pkg/controller/multiarchbuildconfig/multiarchbuildconfig.go

```diff
@@ -120,41 +113,43 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, logge
 	return nil
 }
 
-func (r *reconciler) handleMultiArchBuildConfig(ctx context.Context, mabc *v1.MultiArchBuildConfig) error {
+func (r *reconciler) handleMultiArchBuildConfig(ctx context.Context, mabc *v1.MultiArchBuildConfig) (resultErr error) {
+	var mutateFn func(mabcToMutate *v1.MultiArchBuildConfig)
+
+	defer func() {
+		if mutateFn == nil {
```

Comment:
We should never have this case.

Reply:
See https://github.com/openshift/ci-tools/pull/3676#discussion_r1381690914

Reply:
I mean that we will never have a nil mutateFn.